include/emp/web/Input.hpp

```diff
@@ -292,18 +294,22 @@ namespace web {
 
     /// Update the min
     Input & Min(const std::string & in_m) { Info()->UpdateMin(in_m); return *this; }
+    /// Update the min
```

Comment:
might be a good use case for doxygen member groups? although these might be a bit heavy for just one duplicated comment

Reply:
https://www.doxygen.nl/manual/grouping.html#memgroup

Reply:
I think I'm not going to worry about it here, but that's a super cool feature and I am glad to know about it for the future! That said, I think a hazard of the more subtle doxygen directives like that is that since they look like comments it's easy to delete them by accident without realizing they're meaningful (we've lost a number of ///@cond s over the years).